metabox/metabox/scenarios/config/test_manifest.py

```diff
@@ -0,0 +1,207 @@
+# This file is part of Checkbox.
+#
+# Copyright 2023 Canonical Ltd.
+#
+# Checkbox is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3,
+# as published by the Free Software Foundation.
+
```

Comment:
deleteme

Reply:
Do you mean the pointless empty line or the whole license header?

Reply:
Empty line - completely useless :)